README.md

```diff
@@ -90,202 +67,29 @@ There are many EOS features that do not require your player to have an Epic Game
 
 Efforts will be made to add corresponding support to features as they are added to the Epic Online Services SDK. The table above reflects the features as of November 2023.
 
-# Getting Started
-
-## Prerequisites
+## Supported Platforms
 
-* An Epic Games Account, you can sign up for [here](https://www.epicgames.com/id/register) (_although, most [features](#supported-eos-sdk-features) do not require an Epic Games Account_).
-* A product configured on the [Epic Games Developer Portal](https://dev.epicgames.com/portal/).
-* A Unity project to integrate the plugin into.
+We currently support the following platforms, details of each can be found on our [Supported Platforms](/docs/supported_platforms.md) document:
 
-> [!NOTE]
-> Your system should also satisfy [Unity's system requirements](https://docs.unity3d.com/2021.3/Documentation/Manual/system-requirements.html) as well as the [EOS system requirements](https://dev.epicgames.com/docs/epic-online-services/eos-get-started/system-requirements)
+* Windows
+* Linux
+* macOS
+* Android
+* iOS
+* Nintendo Switch
 
-## Importing the Plugin
+## Importing the Plugin Package
 
 There are two options to install the package:
-* Via a [tarball](#adding-the-package-from-a-tarball) _(easiest to get started quickly)_.
-* From a [git url](#adding-the-package-from-a-git-url) _(this method has the possible advantage of keeping the plugin up-to-date, if that's something that you would prefer)_.
-
-### Adding the package from a tarball
-
-1. Download the latest release tarball, `"com.playeveryware.eos-[version].tgz"` [here](https://github.com/PlayEveryWare/eos_plugin_for_unity/releases).
-
-    > [!WARNING]
-    > Do *not* attempt to create a tarball yourself from the source, unless you know what you are doing with respect to [Git LFS](https://docs.github.com/en/repositories/working-with-files/managing-large-files/configuring-git-large-file-storage).
+* Via a [tarball](/docs/add_plugin.md#adding-the-package-from-a-tarball) _(easiest to get started quickly)_.
```

Comment:
I still believe we should specify that it is a UPM package not just a tarball. Github has attached to each release a source tarball which I would like to ensure that people don't confuse or conflate.

Reply:
As much as I agree with the potential for confusion, the UI within Unity Package Menu specifically refers to it as a tarball. I believe we are pretty clear about how to do this in greater detail on the [add_plugin.md](https://github.com/PlayEveryWare/eos_plugin_for_unity/blob/docs/doc-update/docs/add_plugin.md) document, can you check to make sure those instructions line up with your understanding? They should not have changed but I want to be extra certain.

Reply:
Maybe we could call it a UPM tarball?